pkg/agent/route/route_linux.go

```diff
@@ -115,19 +116,46 @@ func (c *Client) Initialize(nodeConfig *config.NodeConfig, done func()) error {
 func (c *Client) initIPTablesOnce(done func()) {
```

Comment:
maybe the name of this function is not appropriate any more?

Reply:
Removed name, moved to anonymous fn. Same behaviour.